tapir/wirgarten/views/register.py

```diff
@@ -379,7 +386,6 @@ def done(self, form_list, form_dict, **kwargs):
                 form_dict[STEP_BASE_PRODUCT].save(
                     mandate_ref=mandate_ref,
                     member_id=member.id,
-                    is_registration=True,
```

Comment:
@mk2301 this is the parameter error I mentionned in https://matrix.to/#/!KxVWGMwDtwURUCFbOt:matrix.org/$3Dj8EdOi8Ar1G9g5tHNUEuokA28jk2jc-IiIOrft_bo?via=matrix.org